docs/Advanced-Usage/Vitis.rst

```diff
@@ -70,14 +84,15 @@ Bitstream Build
         custom_runtime_config: null
 
 .. Note:: If for some reason the ``buildbitstream`` failed, you can download a pre-built ``xclbin`` here:
-   https://people.eecs.berkeley.edu/~abe.gonzalez/firesim_rocket_singlecore_no_nic.xclbin.
+   https://firesim-ci-vitis-xclbins.s3.us-west-2.amazonaws.com/firesim_rocket_singlecore_no_nic_9e1a21.xclbin
```

Comment:
This is already out of date. We should probably point at only release versions of this in the docs, or point at a script that can intelligently fetch the latest build. 

I guess if we get the URI support sorted for xclbins that will just come for free.

Reply:
I think we should fix the URI support then this becomes easier to handle. But yes, I understand this is out of date... IIRC @t14916 had some `buildbitstream` automation that he wants to bring into the open-source (that should properly update this field).